### DISCUSSION TITLE
Continuous Screen Recording on Devices Without Manual Connection

### DISCUSSION THREAD

**Discussion by bolavefasfas** [Ylianst/MeshCentral#6142] [Q&A]
I'm seeking guidance on setting up automatic screen recording for three of my seven devices. I want the recording to start as soon as the devices are awake till they shut down, without manually connecting to each desktop. How can I achieve this in MeshCentral?

Please dont worry about the space it will take I have a new hdd for this specific task.

**Comment by si458** [comment]
No. 
sessionrecording only records ur actions when u connect to the device
So remote desktop, files and terminal

**Reply by bolavefasfas** [reply]
> No. sessionrecording only records ur actions when u connect to the device So remote desktop, files and terminal

I asked thisbecause I was doing one ssession it was about  3 hours long and 
asap recording was done browser gave error out of memory and recoring did not get saves on pc.

**Reply by si458** [reply]
Did u use the sessionrecording to record server side automatically when you connected to the device or did u use the local recording where u clicked record?

**Comment by hollenal** [comment]
Bumping this up, as I'm also looking for a solution to automatically record (server-side) the desktop session when a machine connects.

**Reply by si458** [reply]
@hollenal unfortunately this is a very low priority feature request, if this feature is urgent/required we are happy to discuss a sponsorship for the feature to be implemented

**Reply by hollenal** [reply]
@si458 sure, I'd be interested to contribute

**Reply by si458** [reply]
@hollenal then plz email myself (check my githib profile page, right hand side)

**Comment by minifiredragon** [comment]
If you can do it without mesh central, I would look at OBS and the OBS-NDI (now called DistroAV) plugin (or even just straight up NDI).
NDI has a bunch of tools for streaming video over the internet, and one of their tools is screen capture.
